app/controllers/users_controller.rb

```diff
@@ -0,0 +1,40 @@
+class UsersController < ApplicationController
+  def index
+    @users = User.all
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(username: params[:username], email: params[:email], password: params[:password])
```

Comment:
Remove commented codes. They make the codebase to be untidy, difficult to read and understand. Use git history if you must view previous codes.